yarn.lock

```diff
@@ -0,0 +1,4846 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Looks like both `yarn.lock` and `package-lock.json` are here? I think it's not recommended to mix versioning systems. In the past I heard npm wasn't as good as yarn for some reason, but now they should be roughly equivalent. Lets stick with just npm instead of yarn (just to minimize the number of things that have to be installed), unless there's a good reason to switch.

Reply:
I used (and advocated for) yarn for a while but switched back to NPM when we stood up SUSHI.  I have no regrets.  NPM has gotten a lot better since yarn emerged on the scene (competition is good) -- and I find that I prefer NPM now.